tests/GlobTrait.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace WP_Rocket\Tests;
+
+trait GlobTrait {
```

Comment:
vfsStream doesn’t work with `glob`. This trait exists to encapsulate replacement functionality for `glob` within the tests.

Reply:
In #2392, I replaced the use of `glob` by SPL iterators instead, since `glob` doesn't work with virtual filesystem. It's also fixing a known issue on some hosts that don't have every flags of `glob` active. What do you think?

Reply:
@Tabrisrp will PR 2392 replace all instances of `glob`?

Reply:
only one, so we will still need the trait for now 👍

Reply:
@Tabrisrp I moved the trait to `wp-media/phpunit`. As we discussed, once we replace `glob()`, then this trait can be deleted and the code in the test removed.